web/styles/widgets.css

```diff
@@ -27,6 +27,11 @@
         margin-right: 4px;
     }
 
+    /* for example in help `?` widget */
+    .spacer {
+        margin: 2px 0 !important;
+    }
+
```

Comment:
Can we use a class name specific to that widget if we only want to fix that.

Reply:
Yeah, that makes sense as `spacer` is too generic.

Reply:
Changed it to `todo-task-spacer`.